cdd_noncovalent_dates_2023_10_18_unfiltered_clean.csv

```diff
@@ -0,0 +1,2147 @@
+Canonical_PostEra_ID,suspected_SMILES,IC50_(µM),IC50_CI_(Lower)_(µM),IC50_CI_(Upper)_(µM),Hill_slope,Curve_class,Semiquant
```

Comment:
This file will need to have its columns permuted so that `suspected_SMILES,Canonical_PostEra_ID` appear in that order, and the `registration_SMILES` appears as well.